src/pages/home/report/ReportActionsView.js

```diff
@@ -399,11 +399,15 @@ class ReportActionsView extends React.Component {
      * @param {Boolean} [shouldResetLocalCount=false] Whether count should increment or reset
      */
     updateLocalUnreadActionCount(shouldResetLocalCount = false) {
-        this.setState(prevState => ({
-            localUnreadActionCount: shouldResetLocalCount
+        this.setState((prevState) => {
+            const localUnreadActionCount = shouldResetLocalCount
                 ? this.props.report.unreadActionCount
-                : prevState.localUnreadActionCount + this.props.report.unreadActionCount,
-        }));
+                : prevState.localUnreadActionCount + this.props.report.unreadActionCount;
+            this.updateUnreadIndicatorPosition(localUnreadActionCount);
+            return {
```

Comment:
Sorry, one last thing. Can we put this on one line? 

Like `return {localUnreadActionCount};`

Reply:
Yes, we can do that!

Reply:
Done